tests/e2e/operationstorm.go

```
@@ -208,4 +211,115 @@ var _ = utils.SIGDescribe("[csi-block-vanilla] Volume Operations Storm", func()
+			scParameters[scParamStoragePolicyID] = profileID
+			// create resource quota
+			createResourceQuota(client, namespace, "100Gi", storagePolicyName)
+		} else if guestCluster {
+			ginkgo.By("CNS_TEST: Running for GC setup")
+			scParameters[svStorageClassName] = storagePolicyName
+			svcClient, svNamespace := getSvcClientAndNamespace()
+			setResourceQuota(svcClient, svNamespace, rqLimit)
+		}
+		if guestCluster {
+			storageclass, err = createStorageClass(client, scParameters, nil, "", "", true, "")
+		} else {
+			scSpec := getVSphereStorageClassSpec(storagePolicyName, scParameters, nil, "", "", false)
+			storageclass, err = client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+		}
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		defer func() {
+			err := client.StorageV1().StorageClasses().Delete(ctx, storageclass.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
+		count := 0
+		for count < volumeOpsScale {
+			ginkgo.By("Creating PVCs using the Storage Class")
+			pvclaims[count], err = fpv.CreatePVC(client, namespace, getPersistentVolumeClaimSpecWithStorageClass(namespace, diskSize, storageclass, nil, ""))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			count++
+		}
+
+		ginkgo.By("Waiting for all claims to be in bound state")
+		persistentvolumes, err = fpv.WaitForPVClaimBoundPhase(client, pvclaims, framework.ClaimProvisionTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		podCount := 0
+		for podCount < volumeOpsScale {
+			ginkgo.By("Creating pod to attach PVs to the node")
+			pvclaim = pvclaims[podCount]
+			podArray[podCount], err = createPod(client, namespace, nil, []*v1.PersistentVolumeClaim{pvclaim}, false, "")
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			podCount++
+		}
+
+		defer func() {
+			err = client.CoreV1().Namespaces().Delete(ctx, namespace, *metav1.NewDeleteOptions(0))
+			if !apierrors.IsNotFound(err) {
```

That was for pod deletion. If pod was deleted later in the test case then pod not found error is ignored in the defer block. It does not apply here.

---

Please check the updated code - it is fixed